AzurePSDrive.psd1

```diff
@@ -29,7 +29,7 @@ Copyright = '(c) Microsoft Corporation. All rights reserved.'
 Description = 'AzurePSDrive provider is a SHiPS based PowerShell provider to simplify navigation and discovery of Azure Resource Manager items'
 
 # Minimum version of the Windows PowerShell engine required by this module
-# PowerShellVersion = ''
+PowerShellVersion = '5.1'
```

Comment:
Agreed. I hope we do not have to deal with CompatiblePSEditions stuff at all.  However `get-module -list` shows up 'desk' instead of 'core' on Linux/CloudShell. That's inconsistent and confusion.

cc @alexandair.

Reply:
Agreed that it's confusing.

I think this is more a PowerShell issue than one with your module.

I've opened https://github.com/PowerShell/PowerShell/issues/7856 to track the issue.

Reply:
Thank you @rjmholt. In this case we can close this PR for now.